Automattic/wp-calypso
# Add duplicate backup notice to composite checkout

#### Changes proposed in this Pull Request

This PR introduces the duplicate backup notices implemented in the current checkout to the composite checkout.

#### Implementation notes

Depends on #44181 

Somehow, `getSitePurchases` seems to return an empty array when composite checkout is enabled, which leads the links in the notice to point to `/me/purchases/` instead of the more specific `/me/purchases/:site-slug/:purchase-id/`. It's good enough for this PR.

#### Testing instructions

- Download the PR
- Follow the instructions in  #44181
- Make sure to activate the composite checkout (append `?flags=composite-checkout-force` to the URL)

#### Screenshots

**Mockup**
<img width="712" alt="Screen Shot 2020-07-16 at 3 00 50 PM" src="https://user-images.githubusercontent.com/1620183/87712846-47ed6f80-c777-11ea-9f03-903b4f8242bf.png">

**Implementation**
<img width="594" alt="Screen Shot 2020-07-16 at 3 14 28 PM" src="https://user-images.githubusercontent.com/1620183/87712863-4de35080-c777-11ea-8077-c47fcf11a501.png">
